src/data/en/index.ts

```diff
@@ -890,6 +891,10 @@ export const loggedInData = {
         scMcExercise: {
           title: 'SC/MC Exercise',
           description: 'Single Choice or Multiple Choice'
+        },
+        fillInTheGap: {
+          title: 'Fill-In-The-Gap Exercise',
```

Comment:
I just checked and actually that's not correct english :)
It probably would be `cloze` or maybe `fill in the blanks`.

Reply:
This would obviously change all the comp and file names too 😬 but it's probably our last chance to do so :)

Reply:
Oh! To be honest, I never checked and just went with the name we were using at serlo. I'll check that and yes, we should change it now instead of later. :smile:

Reply:
It is "fill in the blanks" (like in H5P) now